sync-core/src/main/java/com/cloudant/sync/query/IndexCreator.java

```diff
@@ -281,4 +323,17 @@ private String createIndexIndexStatementForIndexName(String indexName, List<Stri
         return String.format("CREATE INDEX %s ON %s ( %s )", sqlIndexName, tableName, cols);
     }
 
+    private String createVirtualTableStatementForIndex(Index index,
+                                                       List<String> columns,
+                                                       List<String> indexSettings) {
+        String tableName = IndexManager.tableNameForIndex(index.indexName);
+        Joiner joiner = Joiner.on(",").skipNulls();
+        String cols = joiner.join(columns);
+        String settings = joiner.join(indexSettings);
+
+        return String.format("CREATE VIRTUAL TABLE %s USING FTS4 ( %s, %s )", tableName,
+                                                                              cols,
+                                                                              settings);
```

Comment:
We should note this as a comment in this code, and also write something in the documentation for the feature.